packages/shared/styles/components/molecules/SfDevice.scss

```diff
@@ -0,0 +1,117 @@
+.sf-device {
+  &, * {
+    box-sizing: border-box;
+  }
+
+  &__desktop, &__laptop, &__tablet, &__mobile {
+    margin: 0 auto 40px;
```

Comment:
you can use our variables - for `40px` it's `--spacer-xl`. You can find them in `packages/shared/styles/variables` :)